src/main/java/frc/trigon/robot/subsystems/ledstrip/LEDStripCommands.java

```diff
@@ -1,114 +1,51 @@
-                () -> {
-                },
-                ledStrips
-        ).ignoringDisable(true);
-    }
-
-    public static Command getAnimateRainbowCommand(double brightness, double speed, LEDStrip... ledStrips) {
-        return new StartEndCommand(
-                runForEach((strip) -> strip.animateRainbow(brightness, speed), ledStrips),
-                () -> {
-                },
-                ledStrips
+                }
         ).ignoringDisable(true);
     }
 
-    public static Command getAnimateColorFlowCommand(Color color, double speed, LEDStrip... ledStrips) {
-        return new StartEndCommand(
-                runForEach((strip) -> strip.animateColorFlow(color, speed), ledStrips),
-                () -> {
-                },
-                ledStrips
+    public static Command getFlashCommand(Color color) {
+        return new SequentialCommandGroup(
+                getStaticColorCommand(color).withTimeout(LEDStripConstants.FLASHING_TIME),
+                getClearLEDsCommand()
         ).ignoringDisable(true);
     }
 
-    public static Command getAnimateLarsonCommand(Color color, double speed, LarsonAnimation.BounceMode mode, int size, LEDStrip... ledStrips) {
-        return new StartEndCommand(
-                runForEach((strip) -> strip.animateLarson(color, speed, mode, size), ledStrips),
-                () -> {
-                },
-                ledStrips
-        ).ignoringDisable(true);
+    public static Command getBlinkingCommand(Color color) {
+        return new RunCommand(() -> LEDStripConstants.LED_STRIP.blink(color)).withTimeout(LEDStripConstants.BLINKING_INTERVAL).ignoringDisable(true);
     }
 
-    public static Command getAnimateRGBFadeCommand(double brightness, double speed, LEDStrip... ledStrips) {
-        return new StartEndCommand(
-                runForEach((strip) -> strip.animateRGBFade(brightness, speed), ledStrips),
-                () -> {
-                },
-                ledStrips
-        ).ignoringDisable(true);
+    public static Command getRainbowCommand() {
+        return new RunCommand(LEDStripConstants.LED_STRIP::rainbow).ignoringDisable(true);
     }
 
-    public static Command getAnimateSingleFadeCommand(Color color, double speed, LEDStrip... ledStrips) {
-        return new StartEndCommand(
-                runForEach((strip) -> strip.animateSingleFade(color, speed), ledStrips),
-                () -> {
-                },
-                ledStrips
-        ).ignoringDisable(true);
-    }
+    public static Command getThreeSectionColorCommand(Supplier<Color> firstSectionColor, Supplier<Color> secondSectionColor, Supplier<Color> thirdSectionColor) {
+        return new ExecuteEndCommand(
+                LEDStripConstants.LED_STRIP::clearLedColors,
+                () -> LEDStripConstants.LED_STRIP.threeSectionColor(firstSectionColor.get(), secondSectionColor.get(), thirdSectionColor.get())
```

Comment:
This doesn't seem right

Reply:
what doesn't seem right?

Reply:
Why is it an `executeEndCommand`? It doesn't feel right to me. Did you check the logic, not even just this but for it all?